element_calcium_imaging/scan.py

```diff
@@ -234,6 +235,7 @@ class ScanInfo(dj.Imported):
     scan_datetime=null   : datetime  # datetime of the scan
     scan_duration=null   : float     # (seconds) duration of the scan
     bidirectional_z=null : boolean   # true = bidirectional z-scan
+    multi_tiff=True      : boolean   # true = multi-tiff scan. Relevant for Bruker PrairieView
```

Comment:
I'd strongly advise against storing acquisition specific type of information in `scan.ScanInfo` - whether or not the type of file is a big tiff or multiple "small" tiff shouldn't matter and shouldn't be a part of `scan.ScanInfo`